wire/gen_wire_csv

```diff
@@ -51,9 +51,6 @@ funding_locked,8,channel-id,8
 funding_locked,16,announcement-node-signature,64
 funding_locked,80,announcement-bitcoin-signature,64
 funding_locked,144,next-per-commitment-point,33
-update_fee,37
-update_fee,0,channel-id,8
-update_fee,8,feerate-per-kw,4
 shutdown,38
```

Comment:
Hmm, I think this change needs its own commit; I think it's caused by a spec update?